utils.c

```diff
@@ -296,6 +296,20 @@ int janus_mkdir(const char *dir, mode_t mode) {
 	return 0;
 }
 
+gchar* janus_make_absolute_path(const gchar *base_dir, const gchar *path)
+{
```

Comment:
Code style: bracket should be on the same line as the function.
Besides, it should be `gchar *` and not `gchar* `.